### DISCUSSION TITLE
jq

### DISCUSSION THREAD

**Discussion by bbaster** [flet-dev/flet#4371] [Packages] [answered]
https://pypi.org/project/jq/

**Comment by bbaster** [comment]
How would I go about packaging it myself?
Is the package hosting open source?

**Comment by tanmay-bhatgare** [comment]
@bbaster if you are referring, how you can pack a native lib(not written in python entirely) to android wheel, then you should follow these steps, [docs](https://flet.dev/docs/publish/android#prerequisites).

**Reply by bbaster** [reply]
Thank you! That's what I've been looking for.

**Reply by tanmay-bhatgare** [reply]
Consider this as answer and closing discussion!?

**Reply by bbaster** [reply]
As jq is not yet packaged, no.

**Reply by FeodorFitsner** [reply, marked as answer]
`jq` is now available for both iOS and Android.
